### PR TITLE
CXEC-44999-Adyen migrate to Checkout API since Adyen no longer develo…

### PR DESCRIPTION
**Pull request by zeyun123**
migrate to Checkout API since Adyen no longer develops the Classic API.

### CODE REVIEW

**Review by rmcotton-sap — APPROVED** [review]

**Comment by zeyun123** [comment]
for new commitment:
as discussed with Richard ,  the notification for refund and capture . the "success"field always be  "true", no matter refund ok or refund failed , so we should use "eventCode" to identify the result .

**Review by rmcotton-sap — APPROVED** [review]

File: postman/adyen/Hosted-Payment-Page/README.md [hunk 1]
```diff
@@ -73,6 +73,16 @@ The ``integrationId`` and ``configurationId`` values identify the payment integr
 
 You can obtain your merchant ID in the Adyen Dashboard.
 
+**5.apiVersion**
+The Checkout API version of Adyen you want to call. suggestion value is v70 or above.
+
+**Note**:
+
+Since Adyen no longer develops or update old Classic API, OPF current only support Checkout API which Adyen suggested.
```
Comment (rmcotton-sap) [inline postman/adyen/Hosted-Payment-Page/README.md:81]: there is no need to document this. We have no existing customers and will just confuse talking about apis we no longer support.

**Review by rmcotton-sap — CHANGES_REQUESTED** [review]
we need to also apply the same change for the Hosted Fields integration regarding success:true issue.

File: postman/adyen/Hosted-Payment-Page/README.md (hunk 1 above)
Comment (zeyun123) [reply postman/adyen/Hosted-Payment-Page/README.md:81]: ok. I will revert this  NOTE

**Pull request merged**